src/sdk/namespace_s3.js

```diff
@@ -90,14 +90,31 @@ class NamespaceS3 {
         dbg.log0('NamespaceS3.list_objects:', this.bucket, inspect(params),
             'list', inspect(res));
 
+        let bucket_info;
+        try {
+            bucket_info = await object_sdk.read_bucket_sdk_config_info(params.bucket);
+        } catch (err) {
+            dbg.warn('list_buckets: read_bucket_sdk_config_info of bucket', params.bucket, 'got an error', err);
+            // in case the config file was deleted during the bucket list - we will continue
+            if (err.rpc_code !== 'NO_SUCH_BUCKET') throw err;
+        }
+
         return {
             objects: _.map(res.Contents, obj => this._get_s3_object_info(obj, params.bucket)),
             common_prefixes: _.map(res.CommonPrefixes, 'Prefix'),
             is_truncated: res.IsTruncated,
             next_marker: res.NextMarker,
+            object_owner: this.get_object_owner(bucket_info),
         };
     }
 
+    get_object_owner(info) {
```

Comment:
For re-use of code I suggest having a single function in object sdk that brings the info about the bucket owner and on these namespace*.js functions we will add in the future extra implementation per namespace, but for now these functions will look like - 

```
// in every namespace*.js file - 
get_object_owner(object_sdk) {
    // TODO: in the future we will add extra implementation per namespace
    const bucket_owner = object_sdk.get_bucket_owner();
    return {
        name: info.bucket_owner.unwrap(),
        id: info._id,
    };
}
```

```
// in object_sdk.js file - 
get_bucket_owner() {
    .....
}

Reply:
I thought of this. Since the suggestion was to source it from namespace cache. Kept like this. In future if we need namespace specific handling, we can do that.

any thoughts here ? or move to object_sdk ?

Reply:
I just don't like to having the same functionality in multiple places, if one place has a bug we will need to fix it 3 times, I don't see a reason for that..

Reply:
sure.. I will move this to object_sdk